### DISCUSSION TITLE
Is it possible to have more than 1 contract?

### DISCUSSION THREAD

**Discussion by hdevelopergit** [scaffold-eth/scaffold-eth-2#779] [Q&A] [answered]
Hi community!
My name is Hugo and I'm learning blockchain. I am working with this wonderful tool scaffold-eth and I am facing a problem. I wanted to test the inheritance in solidity. I am wondering if it is possible to have more than one contract. Currently I have "yourcontract" but if I want to make inheritance like contract B is YourContract it doesn't work. Any help? Thanks already!

**Comment by technophile-04** [comment, marked as answer]
Are you facing difficulty with deploying multiple contracts from hardhat ? If so you could check out this inheritance [example](https://github.com/scaffold-eth/scaffold-eth-2/commit/1a1c324209b52671f40d7b58720a3997271a6e70#diff-257cc374a4876603ecf985e8702177d10107942b03e68cc0327ad12b527f3289)

**Comment by hdevelopergit** [comment]
Perfect! I was looking for that. Many thanks!
